src/core/plugins/core-factions/shared/interfaces.ts

```diff
@@ -235,6 +235,14 @@ export interface FactionCore {
      * @memberof Faction
      */
     type: string;
```

Comment:
Use a Template Type that can be passed to the Faction interface with a default of the `FactionTypes`.